### PR TITLE
Docs: restructure documentation

### PR DESCRIPTION
**Pull request by sharafian**
As per https://github.com/interledgerjs/ilp-kit/issues/107 . This version of the setup is **not secure** for a permanent ILP Kit instance, due to possible squatting issues. However, it's fine for someone who just wants to try out ILP Kit without doing all that SSL and nginx setup.

### CODE REVIEW

**Comment by emschwartz** [comment]
I think this would be better in a separate quickstart section of the docs rather than mixed in with the instructions for running a permanent kit. My proposal for the readme would be something like this: https://gist.github.com/emschwartz/59453f3358ee2f9b90a650c014839f00

File: README.md [hunk 1]
```diff
@@ -15,150 +15,67 @@ ILP wallet with hosted ledger and connector instances
...
-##### Webfinger
-Webfinger is used to lookup account/user identifiers.
-
-Example request 
-```bash
-curl -X GET
-https://wallet.example/.well-known/webfinger?resource=acct:alice@wallet.example
+# Quick Start
+
+_Prerequisites: Git, Node v6.9.1, `build-essentials` (or XCode command-line tools on OSX)_
```
Comment (emschwartz) [inline README.md]: It would be good to provide a link to documentation for installing the xcode tools

**Comment by sharafian** [comment]
Now depends on https://github.com/interledgerjs/ilp-kit/pull/151 and should be merged at the same time as https://github.com/interledgerjs/ilp-kit-cli

File: README.md [hunk 2]
```diff
@@ -15,150 +15,37 @@ ILP wallet with hosted ledger and connector instances
...
-- [Advanced Mode](#advanced-mode)
-- [Architecture](#architecture)
-  - [Backend (REST API)](#backend-rest-api)
-    - [API docs](#api-docs)
-    - [SPSP](#spsp)
-      - [Webfinger](#webfinger)
-  - [Client](#client)
-    - [Using Redux DevTools](#using-redux-devtools)
-    - [Theme Customization](#theme-customization)
+_Prerequisites: Git, Node v6.9.1, `build-essentials` (or XCode command-line tools on OSX)_
```
Comment (emschwartz) [inline README.md]: I would make each of these links to installation docs

File: bin/localTunnel.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    console.log('Error occurred:', err.message, 'Restarting localtunnel.')
+    connect(port, host)
+  }  
+}
+
+// if the hostname is a subdomain of "localtunnel.me"
+if (process.env.CLIENT_HOST.match(/\.localtunnel\.me$/)) {
+  connect(process.env.CLIENT_PORT, process.env.CLIENT_HOST)
+} else {
+  console.log('Host is not on localtunnel. Exiting with success.')
```
Comment (emschwartz) [inline bin/localTunnel.js]: I don't think it should log anything if you're not using localtunnel

Comment (sharafian) [inline bin/localTunnel.js]: It would say "localtunnel exited with code 0", so I just want to make sure nobody thinks that's a crash or an error

Comment (emschwartz) [inline bin/localTunnel.js]: But if I'm not using localtunnel (i.e. I've set up my own domain) I wouldn't expect anything to happen with localtunnel so I think it should exit silently.

File: bin/localTunnel.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      if (err) {
+        reconnect()
+        return
+      }
+
+      debug('connected localtunnel to', tunnel.url)
+      tunnel.on('close', reconnect)
+      tunnel.on('error', reconnect)
+    })
+  } catch (err) {
```
Comment (emschwartz) [inline bin/localTunnel.js]: Why are you wrapping a function that has a callback in a `try...catch`?

File: docs/ENV.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+##### Required
+
+Name | Example | Description |
+---- | ------- | ----------- |
+`API_HOSTNAME` | `wallet.com` | API public hostname.
+`API_PORT` | `3000` | API private port (used as both public and private port if `API_PUBLIC_PORT` is not specified).
+`API_DB_URI` | `postgres://localhost/wallet` | URI for connecting to a database.
+`API_LEDGER_ADMIN_USER` | `admin` | Ledger admin username.
+`API_LEDGER_ADMIN_PASS` | `pass` | Ledger admin pass.
+`CLIENT_HOST` | `wallet.com` | Publicly visible hostname.
```
Comment (emschwartz) [inline docs/ENV.md]: Why are these called `CLIENT...`? These are public values for the server, not a client

Comment (sharafian) [inline docs/ENV.md]: I think that should be filed an issue, to be put in a breaking version

Comment (sharafian) [inline docs/ENV.md]: I also think all the `_HOST` fields should be unified, so there's less potential for error.

File: bin/localTunnel.js (hunk 4 above)
Comment (justmoon) [inline bin/localTunnel.js]: Wise man say: Just 'cause it has a callback don't mean it don't throw.

File: docs/ENV.md (hunk 5 above)
Comment (justmoon) [inline docs/ENV.md]: There shouldn't be any `_HOST`, `_HOSTNAME`, `_PUBLIC_HTTPS`, `_PUBLIC_PORT` or `_PUBLIC_PATH` fields. Everything should use `*_PUBLIC_URI` like the ledger.

**Comment by sharafian** [comment]
@emschwartz @justmoon @vhpoet This PR is working on the latest master. There are a few issues in the UI (payments don't group together, there isn't a transition when a payment appears in history, and the lag in input fields can sometimes erase what you've written), but overall it's functional. I think we should merge it soon, so we have this feature for future workshops and hackathons.

**Comment by sharafian** [comment]
It also wipes history on reload, and the settlement_methods page causes sqlite syntax errors, so those are problems too.

File: README.md [hunk 6]
```diff
@@ -15,151 +15,37 @@ ILP wallet with hosted ledger and connector instances
...
-
-Example request 
-```bash
-curl -X GET
-https://wallet.example/.well-known/webfinger?resource=acct:alice@wallet.example
+```sh
+$ git clone https://github.com/interledgerjs/ilp-kit
+$ cd ilp-kit
+$ npm install
+$ npm run build
```
Comment (justmoon) [inline README.md]: `npm run build` is currently automatically run after `npm install`, so you could leave it out here.

File: README.md [hunk 7]
```diff
@@ -15,151 +15,37 @@ ILP wallet with hosted ledger and connector instances
...
-      "rel": "https://interledger.org/rel/sender/pathfind",
-      "href": "https://red.ilpdemo.org/api/payments/findPath"
-    },
-    {
-      "rel": "https://interledger.org/rel/receiver",
-      "href": "https://red.ilpdemo.org/api/receivers/alice"
-    }
-  ]
-}
+Use the default option for every question in the `configure` command,
```
Comment (justmoon) [inline README.md]: Really? If you used the default option for every question, wouldn't your ILP Kit have a name like `localhost` and collide with others? Should `ilp-kit-cli` use some sort of name generator like https://github.com/Atrox/haikunatorjs?

Comment (sharafian) [inline README.md]: https://github.com/interledgerjs/ilp-kit-cli/pull/24 I can switch this to use haikunator

**Comment by justmoon** [comment]
Closing for now - the UI isn't quite there. @vhpoet will improve it and open a new PR.

**Pull request closed without merging**